tests/tracking/test_client.py

```diff
@@ -292,7 +292,7 @@ def test_create_model_version_nondatabricks_source_no_runlink(mock_registry_stor
     assert model_version.run_id == "runid"
     # verify that the store was not provided a run link
     mock_registry_store.create_model_version.assert_called_once_with(
-        "name", "source", "runid", [], None
```

Comment:
Would probably be good to get a super basic test in here that uses the description in the API call (currently seems like they all don't call with description)